src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,287 @@
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: bulkWrite
+                OperationCommand:
+                  WriteOperations:
+                    { ^Parameter: { Name: "Operations", Default: "unset" } }
+  - TemplateName: NormalWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: "unset" } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: insertOne
+                OperationCommand:
+                  Document: *Document
+
+Actors:
+  # This loads about 35 GB of data to warm up the DB
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: Warmup
+        Threads: 20
+        OnlyActiveInPhase: [0, 4, 9, 13, 18, 22, 27]
+        Duration: 2 minutes
+        Operations: *WriteOps100x
+
+  - Name: BuildIndexActor
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [5, 14, 23, 28]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *DbName
+          Operations:
+            - OperationMetricsName: BulkBuildColumnStoreIndex
+              OperationName: RunCommand
+              OperationCommand:
+                createIndexes: *CollectionName
+                indexes:
+                  - key: { sensorId: 1, timestamp: -1 }
+                    name: SensorAndTimestamp
+                  - key: { timestamp: -1 }
+                    name: Timestamp
+
+  - Name: QuiesceActor
+    Type: QuiesceActor
+    Threads: 1
+    Database: *DbName
+    Phases:
+      OnlyActiveInPhases:
+        Active: [3, 8, 12, 17, 21, 26, 31]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+
+  - Name: DeleteCollectionActor
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2, 7, 11, 16, 20, 25, 30]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *DbName
+          Operations:
+            - OperationName: RunCommand
+              OperationCommand:
+                drop: *CollectionName
+
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: BulkWriteSingleConnectionNoIndex
+        Threads: 1
+        OnlyActiveInPhase: [1]
+        Duration: 1 minute
```

Comment:
suggestion: make this duration a constant / yml reference and reuse it.

Is 1 minute correct or should we have a longer duration?

Reply:
Based on your question, I've looked at the variability of the results [here](https://github.com/10gen/product-perf-experimentations/blob/master/users/simon.eismann/PERF-4368/analysis.ipynb). Most measurements are sufficiently stable within 1 minute, but I've made the test longer for the non-bulkload writes as they show a higher variation.